tmt/cli.py

```diff
@@ -546,8 +546,11 @@ def import_(
 @click.pass_context
 @name_filter_condition_long
 @click.option(
-    '--nitrate', is_flag=True,
-    help='Export test metadata to Nitrate.')
+    '-h', '--how', metavar='METHOD',
+    help='Use specified method for export (nitrate or polarion).')
+@click.option(
+    '--project-id', default='RedHatEnterpriseLinux7',
```

Comment:
technically this is not needed for most usages (if test exists in polarion it will be found and this option will not be used), I can remove the default but this project is readonly anyway so it would fail either way